cmd/handlers.c

```diff
@@ -178,6 +181,32 @@ cmdGetTime(CmdPacket *cmd) {
   return;
 }
 
+// Command handler for IP information
+static void ICACHE_FLASH_ATTR
+cmdGetWifiInfo(CmdPacket *cmd) {
+  CmdRequest req;
+
+  cmdRequest(&req, cmd);
+  if(cmd->argc != 0 || cmd->value == 0) {
```

Comment:
Mhh, I have to look what  has been done elsewhere, but I'm not sure why this should return anything if no callback has been passed.